### PR TITLE
add main admin page new-streetcode styles

### PR DESCRIPTION
**Pull request by Dobriyr**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Kotusyk — COMMENTED** [review]
Good job!

File: src/features/AdminPage/NewStreetcode/MainNewStreetcode.styles.scss [hunk 1]
```diff
@@ -4,15 +4,99 @@
...
+
+        .button-margin{
+            &-vertical{
+                margin: 12px 0px;
+            }
+            &-left{
+                margin-left: 5px;
+            }
+            &-right{
+                margin-right: 5px;
```
Comment (MaksBrat) [inline src/features/AdminPage/NewStreetcode/MainNewStreetcode.styles.scss:96]: and here

File: src/features/AdminPage/PageBar/PageBar.styles.scss [hunk 2]
```diff
@@ -7,23 +7,29 @@
...
     @include mut.flexed(column, $align-items: flex-start, $gap: 10px);
-    margin-top: 20px;  
-    margin-left: 30%;
-    min-width: 100px;
+    position: fixed;
 
-    span {
+    .Link {
+        color:black;
+        margin-bottom: 10px;
```
Comment (MaksBrat) [inline src/features/AdminPage/PageBar/PageBar.styles.scss:22]: same

**Review by MaksBrat — APPROVED** [review]

**Comment by MaksBrat** [comment]
Great job!

**Pull request closed without merging**